src/combobox/ComboBox.ts

```diff
@@ -394,7 +396,7 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 			v('button', {
 				'aria-controls': menuId,
 				bind: this,
-				classes: this.classes(css.arrow),
+				classes: this.classes(css.trigger, iconCss.icon, iconCss.chevronDown),
```

Comment:
I definitely like the intellisense that comes with `iconCss`, and I think it should be used like this when users are creating their own VNodes. My only concern is for edge cases within themed widgets. The first is purely semantic: if someone wants to change one icon to a different icon, they could theoretically theme `iconCss` so `.chevronDown` is a plus sign, but it seems messy. The other problem is if anyone ever wants to theme two separate uses of the same icon differently. Is that possible with the current usage?